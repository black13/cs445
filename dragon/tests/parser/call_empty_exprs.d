program example(input, output);
var x: char;
begin
    x := getchar()
end.
