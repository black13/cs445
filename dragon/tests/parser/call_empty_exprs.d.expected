PROGRAM `example` WITH ARGS:
  `input`
  `output`
AND VARIABLES:
  DECLARATION WITH TYPE:
    CHAR
  OF NAMES:
    `x`
AND SUBPROGRAMS:
DOES:
  ASSIGN
    x
  TO
    APPLY `getchar`
    TO ARGS:
-- done dumping ast --
